docs/getting-started/flatbuffers.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Read, write, and visualize MCAP files containing Flatbuffer data.
+---
+
+# Flatbuffers
+
+## Read and write MCAP
+
+If you're starting from scratch, you can write code that allows you to **write your Flatbuffer data to MCAP files** and subsequently **read your Flatbuffer data from your MCAP files**.
+
+### Examples
+
+- [TypeScript](https://github.com/foxglove/mcap/tree/main/typescript/examples/flatbufferswriter) - [Writing Foxglove schemas to MCAP file](https://github.com/foxglove/mcap/blob/main/typescript/examples/flatbufferswriter/scripts/main.ts)
+
+## Inspect MCAP
```

Comment:
need to merge flatbuffer support for MCAP CLI before saying this, as i'm pretty sure it can't `cat --json` for flatbuffer messages.

Reply:
oh yeah, good callout -- removed